angular-client/src/services/socket.service.ts

```diff
@@ -24,6 +24,7 @@ export default class SocketService {
     this.socket.on('message', (message: string) => {
       try {
```

Comment:
there should be some more comments throughout this method so it's easier to follow what lines are doing what

Reply:
receiveData